tools/perfscale-load-generator/object/namespace.go

```diff
@@ -32,7 +32,12 @@ import (
 	"kubevirt.io/client-go/log"
 )
 
-func CreateNamespace(virtCli kubecli.KubevirtClient, name, scenarioLabel, uuid string) error {
+var createdNamespaceCache = map[string]bool{}
```

Comment:
Is it really needed? It feels like some optimization that can backfire easily - the cache is not updated for the namespace that were deleted, so we might not be creating namespaces that should be created.